unstructured/partition/common.py

```diff
@@ -124,3 +129,24 @@ def convert_office_doc(input_filename: str, output_directory: str, target_format
 - Mac: https://formulae.brew.sh/cask/libreoffice
 - Debian: https://wiki.debian.org/LibreOffice""",
         )
+
+
+def exactly_one(args: List[Any], names: List[Any]) -> None:
+    # Verify correct usage of `exactly_one`
+    if len(args) != len(names):
+        raise Exception(
+            "`exactly_one` is not used correctly."
+            " The list of arguments must be equal in length to the list of argument names.",
+        )
+    if len(args) <= 1:
+        raise Exception(
+            "`exactly_one` is not used correctly."
+            " At least two arguments must be used,"
+            " otherwise there is no need to use `exactly_one`.",
+        )
```

Comment:
This is indeed just to verify that future maintainers use the method correctly, rather than users of the partitioning functions. I will remove this conditional.